src/plugins/intel_gna/src/transformations/pwl/configuration_log.cpp

```diff
@@ -0,0 +1,74 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "configuration_log.hpp"
+
+#include "common.hpp"
+#include "ie_common.h"
+#include "openvino/opsets/opset11.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+
+namespace ov {
+namespace intel_gna {
+namespace pass {
+namespace pwl {
+
+const std::string ConfigurationLog::kActivationName = "Log";
+constexpr const double ConfigurationLog::kLowerBound;
+constexpr const double ConfigurationLog::kUpperBound;
+constexpr const double ConfigurationLog::kMinValue;
+constexpr const double ConfigurationLog::kMaxValue;
+constexpr const double ConfigurationLog::kPerformanceErrorPercentage;
+constexpr const double ConfigurationLog::kAccuracyErrorPercentage;
+constexpr const double ConfigurationLog::kBreakBound;
+constexpr const size_t ConfigurationLog::kMaxIterations;
+
+double ConfigurationLog::FunctionLog::get_value(double x) const {
+    return std::log(x);
+}
+
+double ConfigurationLog::FunctionLog::get_first_derivative(double x) const {
+    return 1.0 / x;
```

Comment:
it's public function: what assures x>0?

Reply:
This is inherited from old code and it assumes that +infinity/-infinity  will be assigned.

https://en.cppreference.com/w/cpp/language/operator_arithmetic

"dividing a non-zero number by ±0.0 gives the correctly-signed infinity and [FE_DIVBYZERO](https://en.cppreference.com/w/cpp/numeric/fenv/FE_exceptions) is raised"